VL.Data/Services/BookService.cs

```diff
@@ -25,27 +28,20 @@ public async Task<List<Book>> GetBooksAsync()
             return await _applicationDbContext.Book.FindAsync(id);
         }
 
-        public async Task<Book?> UpdateBookAsync(Book book)
+        public async Task<Book?> UpdateBookAsync(UpdateBookDto book)
```

Comment:
Sure `UpdateBookDto` as a model name makes sense in this one method, but loses traction everywhere else use `BookDto`

Create
Get
etc..